spacy_llm/tasks/templates/spancat.v3.jinja

```diff
@@ -0,0 +1,64 @@
+You are an expert Named Entity Recognition (NER) system.
```

Comment:
For spancat, should we call this differently, maybe at least drop the "named" bit? I'm not sure though, maybe NER + overlapping allowed is fine.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/180/commits/1c269e04afea70397dbfcf558f4355b6b6894e24.